app/controllers/v1/sessions_controller.rb

```diff
@@ -30,7 +32,15 @@ def new
       raise Common::Exceptions::RoutingError, params[:path] unless REDIRECT_URLS.include?(type)
 
       StatsD.increment(STATSD_SSO_NEW_KEY,
-                       tags: ["context:#{type}", "forceauthn:#{force_authn?}", VERSION_TAG])
+                       tags: ["context:#{type}", VERSION_TAG])
+      if force_authn?
```

Comment:
Since we’re tracking STATSD_SSO_NEW_FORCEAUTH and STATSD_SSO_NEW_INBOUND, do we still want to track STATSD_SSO_NEW_KEY as a separate metric? I see that STATSD_SSO_NEW_KEY used to have a tag of forceauthn, but no longer does.

Could you also add specs for the new STATSD_SSO_NEW_FORCEAUTH and STATSD_SSO_NEW_INBOUND metrics please?

Reply:
All 3, or at least the ability to track the 3 scenarios, is going to be needed for our reporting.  We want to know when
a) a user initiates a new authentication request
b) a user initiates a new authentication request using forceAuthN
c) a user initiates a new authentication request using inbound SSOe (auto login)

`b` and `c` are optional, but `a` will happen every time.  I think we can do this with 1 stat and 2 tags or just 3 different stats.  I'm not sure what the preferred way for measuring this is, but I'm open to suggestions.

Reply:
Ah ok, thanks for the explanation. I think the three stat approach is fine.  My only other comment is just some additional specs for the inbound and forceauth metrics. Thanks!

Reply:
sure thing.  the specs have been updated here https://github.com/department-of-veterans-affairs/vets-api/pull/4319/files#diff-b6ff071088096f7d49f49f8b428ab449L86-R137